Repositories/ProjectRepository.cs

```diff
@@ -159,6 +159,7 @@ public override async Task<Project> FindAsync(int id)
                                     .Where(s => s.Id == id)
```

Comment:
More a note for later, but do we need all these images in each of these functions? Maybe we should take a look at that with performance. But maybe i'm wrong and they aren't needed idk.